products/plesk.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Plesk
+category: server-app
+iconSlug: plesk
+permalink: /plesk
+versionCommand: plesk version
+releasePolicyLink: https://www.plesk.com/lifecycle-policy/
+eolColumn: End Of Life
+releaseColumn: true
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "18.0.50"
```

Comment:
My first thought was to say that the cycle should be only `18`. But given the new release policy adopted for Plesk 18.x I understand why you made such choice.

https://docs.plesk.com/release-notes/obsidian/change-log seems to list all Plesk 18 releases with their date, so it should be possible to write a custom automation script that would :

- update automatically tiny releases (such as 18.0.48.1, 18.0.48.2...),
- let us know when there are new releases (such as 18.0.51, 18.0.52...).

Unfortunately we can't add new release cycles automatically for the time being, we can only update the existing one.

Reply:
>  it should be possible to write a custom automation script 
I can even help to write it, but I'm not very familiar with the project yet. If you have the examples, please point out on them.

> we can't add new release cycles automatically
I'm not sure how to address this. In one hand, we support two releases simultaneously, so I want to highlight the fact that not only the last release is supported. On the other hand, we can keep the information only about latest release in this MD file (cons: no history, no info about one more supported version of the product).

Reply:
> I can even help to write it, but I'm not very familiar with the project yet. If you have the examples, please point out on them.

Scripts are located in https://github.com/endoflife-date/release-data. Every 6 hours the scripts are ran to retrieve new releases an store them as JSON in https://github.com/endoflife-date/release-data/tree/main/releases.

And every day, the releases in product's markdown files are updated using the data stored in the JSON files. That's done in [latest.py](https://github.com/endoflife-date/endoflife.date/blob/master/_auto/latest.py).

I think the script for plesk should be similar to what's done in [cos.py](https://github.com/endoflife-date/release-data/blob/main/src/cos.py).

Reply:
> I'm not sure how to address this. In one hand, we support two releases simultaneously, so I want to highlight the fact that not only the last release is supported. On the other hand, we can keep the information only about latest release in this MD file (cons: no history, no info about one more supported version of the product).

There is no need to address this one and we can keep the current approach. New releases will be detected every month (you can take a look at https://github.com/endoflife-date/endoflife.date/pull/2550#issuecomment-1438442065 for an example). And someone will have to manually add the new release in the markdown front matter.

Reply:
Automation will be implemented in https://github.com/endoflife-date/release-data/issues/78.